app/web/src/features/profile/ProfileTagInput.tsx

```diff
@@ -195,9 +195,9 @@ export default function ProfileTagInput({
         className={classes.popper}
       >
         <div className={classes.header}>
-          Press 'Enter' to add <br></br>
-          <Link href="mailto:support@couchers.org">Email us</Link> to
-          add a language
+          <Typography>Press 'Enter' to add</Typography>
+          <Link href="mailto:support@couchers.org">Email us</Link> to add a
+          language
```

Comment:
Removed the `<br>`s here and wrapped the "Press 'Enter' ..." in `Typography` instead since:
- they should really be two sentences, so the `<br />` isn't use semantically and the spacing is better dealt with by CSS in this case
- it makes internationalising this slightly easier later on - otherwise, we'd need to bundle the whole message into one long string, which I don't think is right and they should be two messages instead.